aws-k8s-cni.yaml

```diff
@@ -0,0 +1,90 @@
+package:
+  name: aws-k8s-cni
+  version: 1.18.3
+  epoch: 0
+  description: Networking plugin repository for pod networking in Kubernetes using Elastic Network Interfaces on AWS
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aws/amazon-vpc-cni-k8s
+      tag: v${{package.version}}
+      expected-commit: 128a08e41001e81ed317410bd386954022b941af
+
+  - name: Build aws-cni
+    uses: go/build
```

Comment:
Is it possible to have `data.items[]` for readability and to avoid duplication? And then we could add each sub-package to `dependencies.runtime`. In that way, we can test each sub-package individually, [see an example here](https://github.com/wolfi-dev/os/blob/ad85ea73149aab33de83619d0efe215f7a49f7e7/nvidia-container-toolkit.yaml#L57-L61). Wdyt?

Reply:
I was going to say the same! It really helps in terms of readability